### PR TITLE
Add ignore-2.12.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Please, after merge, add 2.11 to the test matrix (I didn't find it)

https://github.com/ansible-collections/overview/issues/45#issuecomment-813821100

### CODE REVIEW

**Review by NilashishC — COMMENTED** [review]
@Andersson007 Thank you for this PR! A trivial changelog fragment is needed for CI to pass. Could you please add that too?

**Comment by Andersson007** [comment]
@NilashishC done, thanks

File: changelogs/fragments/275-add_ignore_txt.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+trivial:
```
Comment (NilashishC) [inline changelogs/fragments/275-add_ignore_txt.yml:1]:
```suggestion
---
trivial:
```

Comment (NilashishC) [reply changelogs/fragments/275-add_ignore_txt.yml:1]: Without the start of document directive, CI will fail.

Comment (Andersson007) [reply changelogs/fragments/275-add_ignore_txt.yml:1]: committed, thanks

**Review by NilashishC — APPROVED** [review]

**Pull request merged**